### PR TITLE
Added and updated frontend routing with comments

### LINKED ISSUE #14: Frontend Routing

### PR DESCRIPTION
**Pull request by ajenaration**
Comments highlight changes to be made upon confirmation, including password functions and file names.

### CODE REVIEW

**Review by Hunajpu — APPROVED** [review]

File: app/__init__.py [hunk 1]
```diff
@@ -37,35 +37,88 @@ def password(self, password):
...
+
     def __repr__(self):
         return '<User: {}>'.format(self.username)
 
 
 @login_manager.user_loader
 def load_user(user_id):
     return Employee.query.get(int(user_id))
-
+    # Employee??
```
Comment (truongnguyenlinh) [inline app/__init__.py]: Thanks for catching, it's been updated.

File: app/__init__.py [hunk 2]
```diff
@@ -37,35 +37,88 @@ def password(self, password):
...
+@app.route("/health")
+def health():
+    return ""
 
 if __name__ == "__main__":
-    app.run()
\ No newline at end of file
+    app.run()
+
+@app.route("/register", methods=("GET", "POST"))
```
Comment (truongnguyenlinh) [inline app/__init__.py:73]: Please add this route before `if __name__ == "__main__": app.run()`

Comment (truongnguyenlinh) [reply app/__init__.py:73]: @nanonite9

**Review by truongnguyenlinh — CHANGES_REQUESTED** [review]

**Review by truongnguyenlinh — APPROVED** [review]
LGTM

**Pull request merged**